.github/workflows/test-dbt-installation-main.yml

```diff
@@ -0,0 +1,73 @@
+# **what?**
+
+# **why?**
+
+# **when?**
+# This workflow will run on a schedule every night and also can be
+# manually invoked.
+
+name: Dbt Installation Tests
```

Comment:
```suggestion
name: dbt Installation Tests
```

Reply:
Fixed in 68329ec7d43fc6ebf807b83ba9196f303058fb74